query_tests/generate_rnd_queries.py

```diff
@@ -74,23 +89,36 @@
 IMPORT_DATA = "COPY benchmarks.query_tests FROM 's3://crate-stresstest-data/query-tests/*.json'"
 
 
+def expr_with_operator(column, provider, inner_type, dims):
+    val = provider()
+    if inner_type in ('string', 'ip'):
+        val = f"'{val}'"
+    operators = OPERATORS_BY_TYPE.get(inner_type)
+    if not operators or random.randint(1, 20) == 1:
+        return f'{column} IS NULL'
+    else:
+        op = random.choice(operators)
+        if dims:
+            return f'{val} {op} ANY ("{column}")'
+        else:
+            return f'"{column}" {op} {val}'
+
+
 def rnd_expr(data_faker, columns):
-    operators = None
-    while not operators:
+    inner_type = None
+    while not inner_type:
         column = random.choice(list(columns.keys()))
         data_type = columns[column]
         inner_type, *dims = data_type.split('_array')
-        operators = OPERATORS_BY_TYPE.get(inner_type)
-    op = random.choice(operators)
     provider = data_faker.provider_for_column(column, inner_type)
-    val = provider()
-    if inner_type in ('string', 'ip'):
-        val = f"'{val}'"
-    if dims:
-        expr = f'{val} {op} ANY ("{column}")'
+    use_scalar = random.randint(1, 15) == 1
```

Comment:
Similar to haudi's comment https://github.com/crate/crate-benchmarks/pull/136/files#r188545131, I think this should be put into a generic function.